### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by samkristoff** [thingsboard/thingsboard#6634]
**Describe the bug**
Swagger API example for `api/entityView/` includes:

```
"keys": {
    "timeseries": "temperature, humidity",
    ...
```

But the server rejects a request with the timeseries keys as a string with the following error message:

```
{"status":400,"message":"JSON parse error: Cannot construct instance of `java.util.ArrayList` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('temperature, humidity'); nested exception is com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `java.util.ArrayList` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('temperature, humidity')\n at [Source: (PushbackInputStream); line: 1, column: 305] (through reference chain: org.thingsboard.server.common.data.EntityView[\"keys\"]->org.thingsboard.server.common.data.objects.TelemetryEntityView[\"timeseries\"])","errorCode":31,"timestamp":"2022-06-03T18:03:48.344+00:00","subscriptionErrorCode":null,"subscriptionEntry":null,"subscriptionValue":null}
```

Passing the key values as an array of strings works as expected:

```
keys: {
    timeseries: ["temperature", "humidity"]
```

**Your Server Environment**
EC2 using Thingsboard PE AMI
ThingsBoard Version: 3.3.4.1PE OAS3

* Connectivity
  * HTTP

**Comment by SuminDenys** [comment]
Hi @samkristoff,

Thank you for your report! This issue has been confirmed, and a prod ticket PROD-4153 has been created to address it. We are closing this issue for now. If you need to update the status of the prod ticket or have any further questions, please feel free to reopen this issue.

**Comment by sumin-denys** [comment]
Hi @samkristoff,

I am pleased to announce that this issue has been resolved in version 3.8.0.

Please see the following [Pull Request](https://github.com/thingsboard/thingsboard/pull/11422)
